homeassistant/components/evohome/climate.py

```diff
@@ -334,33 +311,44 @@ def preset_mode(self) -> Optional[str]:
         """Return the current preset mode, e.g., home, away, temp."""
         return TCS_PRESET_TO_HA.get(self._evo_tcs.systemModeStatus["mode"])
 
-    async def async_set_temperature(self, **kwargs) -> None:
-        """Do nothing.
+    @property
+    def min_temp(self) -> float:
+        """Return the minimum target temperature of a evohome Controller."""
+        return None
 
-        The evohome Controller doesn't have a target temperature.
-        """
-        return
+    @property
+    def max_temp(self) -> float:
+        """Return the maximum target temperature of a evohome Zone."""
+        return None
+
+    async def async_set_temperature(self, **kwargs) -> None:
+        """Do nothing as evohome Controller don't have target temperatures."""
+        raise NotImplementedError("Evohome Controllers don't have target temperatures.")
```

Comment:
Would it be an idea to forward the command to the master controller if this is called ?

Reply:
This is the master Controller.  It inherits `raise NotImplementedError() from teh Climate class`, which is correct, but I wanted to have a better error message.

Reply:
... there is no target temp stuff in the UI - it was for the service call. Shall I just take it out as it: 
```python
self.supported_features & SUPPORT_TARGET_TEMPERATURE == False
```

Reply:
Yes, I think leave it in.